ls1intum/Artemis
# `Development`: Update client and server dependencies

### Checklist
#### General

### Motivation and Context
Update client and server dependencies. There are no major updates for production dependencies, so there should be no breaking changes. However, a couple of third party vulnerabilities were fixed.

### Steps for Testing
Simply browse around some important screens and double check they still work correctly

### Review Progress

#### Code Review
#### Manual Tests
#### Exam Mode Test
